pkg/recipes/terraform/execute.go

```diff
@@ -89,20 +82,20 @@ func (e *executor) Deploy(ctx context.Context, options Options) (*tfjson.State,
 		return nil, err
 	}
 
-	// Create Working Directory
-	workingDir, err := createWorkingDir(ctx, options.RootDir)
+	// Create a new instance of tfexec.Terraform with current Terraform installation path
+	tf, err := NewTerraform(ctx, options.RootDir, execPath)
```

Comment:
Could working in the Terraform installation path cause clashes with other files that will take place in the same directory?

Before we were creating a working directory (**workingDir**) in the root directory, now we will not create a working directory but will work in the execution path (**execPath**) -- the working directory and execution path will be the same. Would that cause any problems?

Reply:
We are still creating working directory inside root directory https://github.com/radius-project/radius/blob/2dde25c6cfac1c54a5fe024c6a71f14d8420da5a/pkg/recipes/terraform/types.go#L67, this PR is just moving creation of working directory closer to initialization of terraform-exec. Did you mean something else?